src/main/java/org/openrewrite/java/testing/jmockit/JMockitMockUpToMockito.java

```diff
@@ -0,0 +1,230 @@
+                    Statement s = m.getParameters().get(i);
+                    if (s instanceof J.VariableDeclarations) {
+                        J.VariableDeclarations param = (J.VariableDeclarations) s;
+                        if (param.getType() instanceof JavaType.Primitive) {
+                            switch (param.getType().toString()) {
+                                case "int":
+                                    sb.append("anyInt()");
+                                    break;
+                                case "long":
+                                    sb.append("anyLong()");
+                                    break;
+                                case "double":
+                                    sb.append("anyDouble()");
+                                    break;
+                                case "float":
+                                    sb.append("anyFloat()");
+                                    break;
+                                case "short":
+                                    sb.append("anyShort()");
+                                    break;
+                                case "byte":
+                                    sb.append("anyByte()");
+                                    break;
+                                case "char":
+                                    sb.append("anyChar()");
+                                    break;
+                                case "boolean":
+                                    sb.append("anyBoolean()");
+                                    break;
+                            }
+                        } else {
+                            String paramType = param.getTypeAsFullyQualified().getClassName();
+                            sb.append("nullable(").append(paramType).append(".class").append(")");
+                        }
+                    }
+                    if (i < m.getParameters().size() - 1) {
+                        sb.append(", ");
+                    }
+                }
+            }
+            sb.append(")").append(")");
+            sb.append(".thenAnswer(invocation -> {");
+            for (Statement s : m.getBody().getStatements()) {
+                sb.append(s.printTrimmed());
+                sb.append(";");
+            }
+            sb.append("});");
+        }
+
+        @Override
+        public J.MethodDeclaration visitMethodDeclaration(J.MethodDeclaration methodDeclaration, ExecutionContext ctx) {
+            J.MethodDeclaration md = super.visitMethodDeclaration(methodDeclaration, ctx);
+            if (md.getBody() == null) {
+                return md;
+            }
+
+            // Create the new statement
+            J.Block body = md.getBody();
+            List<String> shouldClose = new ArrayList<>();
+            for (Statement statement : body.getStatements()) {
+                if (!(statement instanceof J.NewClass)) {
+                    continue;
+                }
```

Comment:
We should likely also check the type of J.NewClass through TypeUtils to ensure it's a `MockUp` before rewriting, as folks might have other unrelated inline classes in the same file.

Reply:
Sure, I missed this one